Dockerfile

```diff
@@ -1,14 +1,20 @@
 FROM golang:1.13.4 AS builder
 WORKDIR /easi/
 COPY . .
-
 RUN  CGO_ENABLED=0 GOOS=linux go build -a -o bin/easi ./cmd/easi
 
 FROM alpine:3
 RUN apk --no-cache add ca-certificates
 WORKDIR /easi/
 COPY --from=builder /easi/bin/easi .
 
+ARG ARG_APPLICATION_VERSION
+ARG ARG_APPLICATION_DATETIME
+ARG ARG_APPLICATION_TS
+ENV APPLICATION_VERSION=${ARG_APPLICATION_VERSION}
```

Comment:
I don't know if these should be set when testing locally or not. If so, should the `.envrc*` files be updated?

Reply:
Good question. Since the variables are supposed to correspond to a build workflow time and version, I'm not sure what to populate them with, or to what end. The tests I wrote should pass even with these variables being unset: the health check would just return a zero-length string.